### ISSUE TITLE
Question about game map on Windows/Linux.

### ISSUE THREAD

**Issue by BaiChunhui-9803** [cpp-sc2/cpp-sc2#126]
Hello.
I run the same bot under the same game map on both Windows and Linux system. But only on Windows can I get the correct results. I found that on Linux, enemy units haven't been observed, which had caused self units to wander aimlessly. 
So I want to know what might be the cause of this problem.

**Comment by alkurbatov** [comment]
Hi @BaiChunhui-9803.
Usually there are no difference between windows and linux from the api pespective.
The only thing I can guess now is that you compiled your bot with latest SC2 Version value. Currently, we have different unit types constants for windows and linux as on windows typically the latest game version is used while on linux we are stick with much older 4.10.2 (or 4.10.0). 

Thus first please verify what version was used in both cases. More details about version are here:
https://github.com/cpp-sc2/cpp-sc2/blob/master/docs/building.md#compilation-options

It is hard to tell what exactly happend without source code. So, if the version values are correct, please, provide some sample code to reproduce the problem.

**Comment by BaiChunhui-9803** [comment]
I think I've found the reason. I used `DebugKillUnit()` and `DebugCreateUnit()` to reset the game state based on the game state I've observed before by using `Observation()`. Now I observed the previous game state once the game started. On Windows, enemy units can be fully observed at 0 gameloop. However, on Linux, it seems to take extra time.

![1](https://user-images.githubusercontent.com/51870011/190555160-c851e9dc-cc4d-436f-b805-2588c6a70426.jpg)
![2](https://user-images.githubusercontent.com/51870011/190555164-ac6dccc7-a44e-40ea-b051-858280e368eb.jpg)
